tests/QueryType/Update/RequestBuilderTest.php

```diff
@@ -146,6 +146,39 @@ public function testBuildAddXmlMultivalueField()
         );
     }
 
+    public function testBuildAddXmlMultivalueFieldWithEmptyArray()
+    {
+        $command = new AddCommand();
+        $command->addDocument(new Document(['id' => [1, 2, 3], 'text' => []]));
+
+        $this->assertSame(
+            '<add>'.
+            '<doc>'.
+            '<field name="id">1</field>'.
+            '<field name="id">2</field>'.
+            '<field name="id">3</field>'.
+            '</doc>'.
+            '</add>',
+            $this->builder->buildAddXml($command)
+        );
+    }
+
+    public function testBuildAddXmlWithEmptyStrings()
```

Comment:
As we had no test so far for that case I added one to the master to get that tested quickly:
https://github.com/solariumphp/solarium/commit/854794fdd2fd9b610f50029009ae05cc2a530f86

Now we have a redundant test ;-)

Reply:
This bug proves that you can never have too many tests. ;-)

Reply:
I changed the `id` to `0` to test for true emptiness. :-D